dask/array/core.py

```diff
@@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
     return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
 
 
+def from_value(value, shape, dtype=None, name=None):
```

Comment:
I would like to release today.  Do we want to squeeze this in?

On Thu, Oct 22, 2015 at 3:28 PM, Stephan Hoyer <notifications@github.com>
wrote:

> In dask/array/core.py
> <https://github.com/blaze/dask/pull/794#discussion_r42814601>:
>
> > @@ -1227,6 +1227,33 @@ def from_array(x, chunks, name=None, lock=False):
> >      return Array(merge({name: x}, dsk), name, chunks, dtype=x.dtype)
> >
> >
> > +def from_value(value, shape, dtype=None, name=None):
>
> Yeah, I think from_imperative is probably the best choice. A little long
> for my taste, but it's pretty clear value is too ambiguous to be a good
> name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/blaze/dask/pull/794/files#r42814601>.
>